src/Infrastructure/Controller/Regulation/DownloadRegulationController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Regulation;
+
+use App\Application\QueryBusInterface;
+use App\Application\Regulation\Query\GetRegulationOrderRecordSummaryQuery;
+use App\Domain\Regulation\Specification\CanOrganizationAccessToRegulation;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\HeaderUtils;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Routing\Requirement\Requirement;
+
+final class DownloadRegulationController extends AbstractRegulationController
+{
+    public function __construct(
+        QueryBusInterface $queryBus,
+        CanOrganizationAccessToRegulation $canOrganizationAccessToRegulation,
+        Security $security,
+        private readonly \Twig\Environment $twig,
+        private readonly string $projectDir,
+    ) {
+        parent::__construct($queryBus, $security, $canOrganizationAccessToRegulation);
+    }
+
+    #[Route(
+        '/regulations/{uuid}/download',
+        name: 'app_regulation_download',
```

Comment:
Ajouter l'extension pour indiquer le format de fichier qui va être télécharger ?

Par ailleurs, partout on parle "d'export" et on utilise un template `export.md`, ne vaut-il pas mieux aussi utiliser ça dans l'URL et le nom du controller ?

```suggestion
final class ExportRegulationController extends AbstractRegulationController
{
    public function __construct(
        QueryBusInterface $queryBus,
        CanOrganizationAccessToRegulation $canOrganizationAccessToRegulation,
        Security $security,
        private readonly \Twig\Environment $twig,
        private readonly string $projectDir,
    ) {
        parent::__construct($queryBus, $security, $canOrganizationAccessToRegulation);
    }

    #[Route(
        '/regulations/{uuid}/export.docx',
        name: 'app_regulation_export',
```

Reply:
J"utilise le paramètre Symfony _format => https://symfony.com/doc/current/routing.html#special-parameters
Il pourra être évolutif si demain, on prends aussi en compte du PDF.